packages/xdl/src/start/startAsync.ts

```diff
@@ -153,7 +152,7 @@ export async function stopAsync(projectRoot: string): Promise<void> {
   try {
     const result = await Promise.race([
       stopInternalAsync(projectRoot),
-      new Promise(resolve => setTimeout(resolve, 2000, 'stopFailed')),
+      new Promise(resolve => setTimeout(resolve, 5000, 'stopFailed')),
```

Comment:
Why is this increased? 5 seconds is a lot of time for something that should be instant.

Reply:
Partially a debug artifact and partially to accommodate the extra API call on bad networks. I agree that the original timeout should be enough though!